frontend/components/Site/Home/TopSection.tsx

```diff
@@ -28,25 +38,22 @@ const TopSection = () => (
           align-items: center;
           padding: 90px 20px;
         }
-      }
 
-      .home-section-top h1 {
-        margin-bottom: 30px;
-        text-align: center;
-      }
-
-      @media (min-width: 960px) {
         .home-section-top h1 {
           margin-bottom: 0;
-          margin-right: 64px;
           text-align: left;
         }
-      }
-      @media (min-width: 1040px) {
-        .home-section-top h1 {
-          margin-right: 136px;
+
+        :global(.get-started-btn) {
+          margin-left: 10px;
         }
       }
+
+      .home-section-top h1 {
+        margin-bottom: 30px;
```

Comment:
It looks like the `margin-bottom: 30px;` should maybe go up in the `h1 { }` styles above? This would give space between the h1 and button on smaller screens and center the h1 and button horizontally on larger screens

<img src="https://user-images.githubusercontent.com/5829188/86143146-ed7ebe80-baa8-11ea-990f-32d7d26c882e.png" width="200" />

Reply:
Huh? That's weird, I tested all this. Here's how it looks for me:
![image](https://user-images.githubusercontent.com/34203886/86146587-44869280-baad-11ea-9844-088ce772585a.png)

Reply:
![image](https://user-images.githubusercontent.com/34203886/86146655-5cf6ad00-baad-11ea-83de-3a4521feaf4c.png)

Reply:
Ooo I think I figured it out 😃  Lines 53-55 are accidentally overriding the media query styles (42-45) for screens above 960px, which is why the text and button aren't centered (the h1 still has `margin-bottom: 30px;` > 960px instead of 0):

<img src="https://user-images.githubusercontent.com/5829188/86149650-1f941e80-bab1-11ea-9622-4316a66e2273.png" width="400" />

If you move lines 53-55 up into the h1 styles on line 27, it should fix it 😊

Reply:
Ugh I'm still not used to `min-width` breakpoints (vs. `max-width`) and my brain gets confused for some reason. Although I'm still confused because the behavior in your screenshot, I don't see that at any screen width on my side? Unless I'm just going barmy

Reply:
Oh wait I think I was also confused by the screenshots themselves. Your top screenshot had the button up against the title which I can't reproduce on any screen width, but I see the second screenshot alignment issue

Reply:
OK I've taken care of the alignment issue in your second screenshot now, still puzzled by the first screenshot though

Reply:
Sorry I think what happened was I noticed something wasn't working with the margin, and I toggled it in the DevTools, making the smaller screen appear like it was smashed up against the button, but in reality the only issue was the larger screen! It looks perfect now 👌🏼

Reply:
Ah, got it!